tephra-distribution/NOTICE_BINARY

```diff
@@ -6,10 +6,19 @@ The Apache Software Foundation (http://www.apache.org/).
 
 Apache Tephra - notice for binary distribution
 ==============================================
-Apache Tephra includes Guice v3.0 and Guice Assisted Inject v3.0. Tephra
-elects to include this software in this distribution under the Apache License.
-You can obtain a copy of the License at:
-http://apache.org/licenses/LICENSE-2.0
+Apache Tephra includes Guice v3.0.
+Google Guice - Core Library
+Copyright 2006-2011 Google, Inc.
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Apache Tephra includes Guice Assisted Inject v3.0.
+Google Guice - Extensions - AssistedInject
+Copyright 2006-2011 Google, Inc.
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
 
 Apache Tephra includes Guava v13.0.1. Tephra elects to include this software
```

Comment:
most of these lines are still not relevant.

Reply:
@johnament Should I remove these lines? I couldn't find a NOTICE in Guava, Gson, javax.inject JARs.

Reply:
Just because a dependency is apache licensed doesn't mean it will have a NOTICE.  NOTICE is optional, so you shouldn't include one if none is present.